### DISCUSSION TITLE
Any plans to implement wireguard support in the future?

### DISCUSSION THREAD

**Discussion by BrodyStone21** [qdm12/gluetun#540] [Ideas]
Title pretty much. It seems wireguard is faster for me for P2P connections and I was just curious if you were planning to implement this.

**Comment by qdm12** [comment]
Subscribe to #134

What provider are you using?

I'm actually working on it right now, implementing it for Mullvad for now.

**Reply by BrodyStone21** [reply]
Sweet, thanks! I'm using PIA.

**Reply by qdm12** [reply]
Cool, for now PIA comes as 2nd place after Mullvad. I'll ping you here to test it out if you don't mind 😉
